frontendservice/src/views/specificShop.vue

```diff
@@ -127,16 +127,16 @@ export default {
             }).then(function(res) {
               if (res.data.token.shopId == self.shopid) {
                 if (res.data.token.status == "ACTIVE") {
-                  self.flag = true;
+                  self.isTokenInCookie = true;
                   self.tokeninfo = res.data;
                 } else {
                   self.allCookieList.splice(
                     self.allCookieList.indexOf(res.data.token.tokenId),
                     1
                   );
                   Cookie.set("tokenid", JSON.stringify(self.allCookieList));
-                  self.flag = false;
-                  self.statusFlag = true;
+                  self.isTokenInCookie = false;
+                  self.isTokenActive = false;
```

Comment:
Didn't see the entire code, is this intensional?

Reply:
yes priyanka.